site/app/templates/grading/simple/Display.twig

```diff
@@ -306,9 +306,11 @@
                     data-origval="{{ component_score }}"
                     data-maxclamp="{{ max_clamp }}"
                     value="{{ component_score }}"
-                    onclick="this.select()"
+                    onclick="this.dataset.typing || this.select()"
+                    ondblclick="this.dataset.typing = true;"
+                    onblur="this.dataset.typing = false;"
                     onchange=" this.style.backgroundColor = 'var(--default-white)';"
-                    onkeydown="javascript: return ['Backspace','Delete','ArrowLeft','ArrowRight','Period'].includes(event.code) ? true : !isNaN(Number(event.key)) && event.code!=='Space'"
+                    onkeydown="if (event.code === 'Escape') {if (this.dataset.typing) this.dataset.typing = false; else this.blur();} return ['Backspace','Delete','ArrowLeft','ArrowRight','Period'].includes(event.code) ? true : !isNaN(Number(event.key)) && event.code!=='Space'"
```

Comment:
This shouldn't be happening inline.  This should go in `<script>` tags at the bottom of this file, or a `.js` file somewhere.